sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts

```diff
@@ -48,7 +53,9 @@ export class KeycloakService {
    */
   public async getOpenIdConfig(): Promise<OpenIdConfig> {
     try {
-      const response = await axios.get(this.authConfig.openIdConfigurationUrl);
+      const response = await firstValueFrom(
```

Comment:
I got the point of using the `firstValueFrom`, just wondering if using the axiosRef directly would not be acceptable and leave the usage of `rxjs` if it is needed.

Reply:
https://github.com/bcgov/SIMS/pull/2098#discussion_r1260039266